### DISCUSSION TITLE
希望增加“打开服务器文件夹”按钮

### DISCUSSION THREAD

**Discussion by Leonmmcoset** [MCSLTeam/MCSL2#143] [功能建议] [answered]
我想修改一些服务器配置（如：修改spigot.yml）时需要打开资源管理器，很麻烦，希望添加“打开服务器文件夹”按钮。

**Comment by lxhtt** [comment, marked as answer]
already exists in dev version v2.2.3.3.  
Check https://mcsl.com.cn/join-preview.html for more details.  :)
